.github/workflows/clean-up-pr.yml

```diff
@@ -20,20 +20,27 @@ jobs:
       - name: Check Out Repo
         uses: actions/checkout@v3
 
-      - name: Dump Environment Variables
-        run: |
-          cd api && printf "STAGING_PREFIX=$STAGING_PREFIX\n \
-            STAGE_NAME=$STAGE_NAME" > .env
-
-      - name: Remove Serverless stack
-        uses: serverless/github-action@v3.1
+      - name: Setup Node
+        uses: actions/setup-node@v3
         with:
-          args: >
-            -c "cd api && 
-            serverless plugin install --name serverless-dotenv-plugin && 
-            serverless plugin install --name serverless-domain-manager && 
-            serverless remove"
-          entrypoint: /bin/sh
+          node-version: '18'
+          cache: 'npm'
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Build and deploy
```

Comment:
Might want to change the name of this step to indicate it’s for destroying the stack

Reply:
Oops I copy and pasted too hard